File: LinkERP.Entity/INV/LBS_INV_WarehouseNextNumbers.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace LinkERP.Entity.INV
{
    public class LBS_INV_WarehouseNextNumbers : BaseEntity
    {
        public Guid CompanyID { get; set; }
        public Guid WarehouseID { get; set; }
        public string Description { get; set; }
        public string NextNumberPrefix { get; set; }
        public int NextNumber { get; set; }
        public string Name { get; set; }
        public LBS_INV_Warehouse LBS_INV_Warehouse { get; set; }
    }
}

File: LinkERP/ClientApp/src/app/models/inv/lbs-inv-warehouse-next-numbers.ts
import { BaseEntity } from "../base-entity";
import { LbsInvWarehouse } from "./lbs-inv-warehouse";

export class LbsInvWarehouseNextNumbers extends BaseEntity {
    CompanyID:any;
    WarehouseID:any;
    Description:any;
    NextNumberPrefix:any;
    NextNumber:any;
    Name:any;
    Warehouse: any = new LbsInvWarehouse();
}
